static/sidebar/js/libs/analytics.js

```diff
@@ -21,7 +21,7 @@
     };
 
     this.track = function(e,data){
-      if (!e || !data || !isset(window,'_tvpa')) return;
+      if (!e || !data || !_tvpa) return console.log('no data');
```

Comment:
This code will break if `_tvpa` doesn't exist, this is because `_tvpa` is read as a global because it is not declared as an argument or variable in that context.
That's why is required to do some of the followings:

-  `"_tvpa" in window`
-  `"undefined" === typeof _tvpa`
- `window.hasOwnProperty("_tvpa")`